tests/testthat/test-label_operators.R

```diff
@@ -0,0 +1,80 @@
+#------------------------  apply_labels, get_label
+
+LABELS_TEST <- data.frame(
+  Variable <- c("USUBJID", "AVAL", "RACE"),
+  Label <- c("Unique Subject Identifier", "Analysis Value", "Race"),
+  Dataset <- c("ADPC", "ADPC", "ADPC")
+)
```

Comment:
Hey @jimrothstein, I suspect the potential problem can be here due to the use of the assignment operator `<-` in the `data.frame` call. Let me know if this solves it:

```suggestion
LABELS_TEST <- data.frame(
  Variable = c("USUBJID", "AVAL", "RACE"),
  Label = c("Unique Subject Identifier", "Analysis Value", "Race"),
  Dataset = c("ADPC", "ADPC", "ADPC")
)
```